api/types/client.go

```diff
@@ -141,26 +141,32 @@ func (h *HijackedResponse) CloseWrite() error {
 // ImageBuildOptions holds the information
 // necessary to build images.
 type ImageBuildOptions struct {
-	Tags           []string
-	SuppressOutput bool
-	RemoteContext  string
-	NoCache        bool
-	Remove         bool
-	ForceRemove    bool
-	PullParent     bool
-	Isolation      container.Isolation
-	CPUSetCPUs     string
-	CPUSetMems     string
-	CPUShares      int64
-	CPUQuota       int64
-	CPUPeriod      int64
-	Memory         int64
-	MemorySwap     int64
-	CgroupParent   string
-	NetworkMode    string
-	ShmSize        int64
-	Dockerfile     string
-	Ulimits        []*units.Ulimit
+	Tags                 []string
+	SuppressOutput       bool
+	RemoteContext        string
+	NoCache              bool
+	Remove               bool
+	ForceRemove          bool
+	PullParent           bool
+	Isolation            container.Isolation
+	CPUSetCPUs           string
+	CPUSetMems           string
+	CPUShares            int64
+	CPUQuota             int64
+	CPUPeriod            int64
+	BlkioReadBpsDevice   string
+	BlkioWriteBpsDevice  string
+	BlkioReadIOpsDevice  string
+	BlkioWriteIOpsDevice string
```

Comment:
Perhaps drop the `Device` suffix, as it looks redundant and doesn't add any more clarity.